docker/sync_repo

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
-  
+
+set -e
+
 REPO=$1
+BRANCH=$2
```

Comment:
Doesn't hurt to add default for BRANCH
`BRANCH="${2:-master}"` 
or whatever it is

Reply:
Good idea!